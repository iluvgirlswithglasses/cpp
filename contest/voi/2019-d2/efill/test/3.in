42
-4058224 -277482
-4058098 -3911418
-4056654 -3976160
-4056272 -3992020
-4042886 -4055834
-3763884 -4057494
-3443286 -4058200
310836 -4058226
759428 -4058228
3393726 -4058142
3901894 -4056716
3904202 -4056666
4007556 -4052346
4031982 -4046538
4049452 -4039456
4054772 -4030224
4056872 -3968604
4057814 -3872072
4058072 -3399652
4058090 -3312238
4058154 -2987338
4058234 1341920
4058168 2134024
4058050 3461530
4057840 3773318
4057656 4035244
4056192 4043716
4011930 4054556
3983286 4055928
3873938 4057484
3584300 4058176
-639948 4058242
-2540432 4058212
-3966898 4057460
-4026786 4053002
-4032294 4045432
-4052808 4014106
-4054346 3949666
-4055478 3897288
-4057796 3784292
-4057972 3629726
-4058216 2484978
-4058014 -6334042 -4057972 -7545354
