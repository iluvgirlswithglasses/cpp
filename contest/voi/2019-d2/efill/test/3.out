31686645.41
